### ISSUE TITLE
[BUG?] DDG Privacy extension seems to prevent loading gmail

### ISSUE THREAD

**Issue by huuhaa** [duckduckgo/duckduckgo-privacy-extension#127]
I'm not quite sure is this because of using DuckDuckGo Privacy Extension (DDGPE), or what, but I have now faced couble of times issues when login to gmail.

Basicly page doesn't load right and at the bottom theres information that some features could not be load and suggestion to use basic html version instead. Both of times after I go and disable DDGPE and reload page, everything is working fine again.

Strange thing is, that if I go and enable DDGPE again and reload the page (gmail) it's still working. Also after re-enabled I can use gmail without any issues many times after that, at least at same browser session. So this doesn't happen everytime, but seems to be easy to reproduce.

Browser: Vivaldi 1.14.1077.45 with chrome extension.

Haven't yet notice same bahavior with Firefox, but on the other hand I have now used mainly Vivaldi.

**Comment by alohaas** [comment]
@huuhaa apologies for not responding to your feedback sooner. If you're still experiencing this issue with Vivaldi can you please provide a little bit more information on your setup – operating system and version? Screenshots would be helpful as well.

Thank you!

**Comment by huuhaa** [comment]
@alohaas Sems to work fine now, just re-enabled plugin and tried to logout & login to gnail again and everything was working.

I have Vivaldi 1.14.1077.55 (Stable channel) (64-bit) installed at the moment, to  Bunsenlabs / Debian stretch.

I also have Privacy Badger installed and some time ago I noticed at twitter that there was some confilct between those two and bouth developerteam seemed to investgate things. Or do I remember incorrectly. Any way No matter do I remember that correctly and is it cause those issues have been fixed or is it something fixed at Vivaldi, things seems to work now.
